scripts/gui/microbe_editor.mjs

```diff
@@ -341,6 +427,167 @@ function onNextTabClicked(){
     }
 }
 
+function onNameInput(event){
+    if(common.isInEngine()){
+        Leviathan.CallGenericEvent("MicrobeEditorNameChanged", {value: event.target.value});
+    }
+    event.stopPropagation();
+}
+
+function selectCellTab(tab){
+    // Hide all
+    document.getElementById("StructurePanelMid").style.display = "none";
+    document.getElementById("AppearancePanelMid").style.display = "none";
+
+    document.getElementById("StructurePanelTop").classList.remove("Active");
+    document.getElementById("AppearanceButton").classList.remove("Active");
+
+    for(const element of organelleSelectionElements){
+        element.element.style.display = "none";
+    }
+
+    for(const element of membraneSelectionElements){
+        element.element.style.display = "none";
+    }
+
+    // Show selected
+    if(tab == "structure"){
+        document.getElementById("StructurePanelMid").style.display = "block";
+
+        for(const element of organelleSelectionElements){
+            element.element.style.display = "";
+        }
+
+        document.getElementById("StructurePanelTop").classList.add("Active");
+    } else if(tab == "appearance"){
+        document.getElementById("AppearancePanelMid").style.display = "block";
+
+        for(const element of membraneSelectionElements){
+            element.element.style.display = "";
+        }
+
```

Comment:
So the behaviour tab doesn't contain anything? The button should be disabled as you can't switch to it.